docs/intro_notebook.ipynb

```diff
@@ -51,45 +51,50 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "141ebc97",
+   "id": "0921bbef-d089-4960-b624-505172f90d76",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Log in to your W&B account\n",
-    "import weave"
+    "# Set your OpenAI API key\n",
+    "from set_env import set_env\n",
+    "\n",
+    "# Put your OPENAI_API_KEY in the secrets panel to the left 🗝️\n",
```

Comment:
Do we want this notebook to be colab-specific?

Reply:
set_env is general purpose exactly to address this. Do agree the comment is specific.